hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/VectorClockTimestamp.java

```diff
@@ -129,7 +129,7 @@ static VectorClockTimestamp copyVector(VectorClockTimestamp vectorClockTimestamp
         return new VectorClockTimestamp(clocks);
     }
 
-    static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
+    public static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
```

Comment:
What do you mean? I personally prefer my asserts statics so there is no hidden context slipping in.